### PR TITLE
TELCODOCS-948: Merge PAO logs in OCP must-gather

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-948](https://issues.redhat.com//browse/TELCODOCS-948): PAO had a separate must-gather to aid debugging. This will be merged into the OCP must-gather so all PAO and low latency info for nodes will be included in the OCP must-gather. Removing/refactoring instances referring to the old PAO must-gather

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/TELCODOCS-948

Link to docs preview:
- https://59307--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-low-latency-tuning.html#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master
- https://59307--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-create-performance-profiles.html#gathering-data-about-your-cluster-using-must-gather_cnf-create-performance-profiles

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support.adoc [hunk 1]
```diff
@@ -72,26 +45,57 @@ In earlier versions of {product-title}, the Performance Addon Operator provided
...
-<1> The default {product-title} `must-gather` image.
-<2> The `must-gather` image for low latency tuning diagnostics.
+When the command completes, the data is saved in a new directory that starts with `must-gather.local`. This directory is created in your current working directory.
 
 . Create a compressed file from the `must-gather` directory that was created in your working directory. For example, on a computer that uses a Linux operating system, run the following command:
 +
 [source,terminal]
 ----
- $ tar cvaf must-gather.tar.gz must-gather.local.5421342344627712289/ <1>
+$ tar cvaf must-gather.tar.gz must-gather-local.5421342344627712289// <1>
```
Comment (sheriff-rh) [inline modules/cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support.adoc:91]: In the build preview, the `/` is not displayed in the command, but the callout below includes the `/`. If you are attempting to print the `/`, you need a forward slash `\` to escape it: 
```
...-local.5421342344627712289/ \<1>
```

See: https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#escape-with-backslashes

**Review by sheriff-rh — COMMENTED** [review]
One comment, otherwise this looks great!

**Review by jab-rh — COMMENTED** [review]
@rohennes, left a question about rendering. Thanks!

**Review by skrthomas — COMMENTED** [review]
Merge review lgtm!

**Comment by jab-rh** [comment]
@skrthomas @rohennes 

It looks like this still isn't rendering correctly. I don't think `//` is what's intended?

**Pull request merged**